src/foam/nanos/dig/drivers/DigFormatDriver.js

```diff
@@ -274,14 +276,15 @@ foam.CLASS({
       try {
         return dao.put(nu);
       } catch ( ValidationException ve ) {
-        throw new DAOPutException(ve.getMessage(), ve);
+        throw new DAOPutException(ve.getTranslation(), ve);
```

Comment:
why do you need to do this? shouldn't the message be automatically translated in the ValidationException when calling getMessage?